yoti-sdk-impl/src/main/java/com/yoti/api/client/shareurl/policy/SimpleWantedAttribute.java

```diff
@@ -17,15 +20,24 @@
     @JsonProperty("optional")
     private final boolean optional;
 
-    SimpleWantedAttribute(String name, String derivation, boolean optional) {
+    @JsonProperty("accept_self_asserted")
+    private final boolean acceptSelfAsserted;
+
+    @JsonProperty("constraints")
+    private final List<Constraint> constraints;
+
+    SimpleWantedAttribute(String name, String derivation, boolean optional, boolean acceptSelfAsserted, List<Constraint> constraints) {
         this.name = name;
         this.derivation = derivation;
         this.optional = optional;
+        this.acceptSelfAsserted = acceptSelfAsserted;
+
+        // TODO: Check with Mike if this is ok?
```

Comment:
TODO

Reply:
Fixed in https://github.com/getyoti/yoti-java-sdk/pull/138/commits/8bde085bd4cc694e43381e6a29a1ccbf62662768